readme.txt

```diff
@@ -1,19 +1,17 @@
 === Microsoft Azure Storage for WordPress ===
-Contributors: msopentech, 10up, morganestes, stevegrunwell, lpawlik, rittesh.patel
```

Comment:
I prefer to maintain previously-listed contributors unless they've asked to be removed.  Any reason we should NOT add these folks back (as well as any others newly significantly contributing)?